packages/flutter/test/material/page_test.dart

```diff
@@ -297,7 +297,7 @@ void main() {
       ),
     );
     await tester.tap(find.text('PUSH'));
-    expect(await tester.pumpAndSettle(const Duration(minutes: 1)), 2);
+    expect(await tester.pumpAndSettle(const Duration(minutes: 1)), 3);
```

Comment:
What was the reason for this change?

Reply:
Because the hover support adds another frame to the animation.